Controller/ResourceController.php

```diff
@@ -39,11 +46,103 @@ public function __construct(
         $this->registry = $registry;
     }
 
-    public function resourceAction($repositoryName, $path)
+    public function getResourceAction($repositoryName, $path)
     {
         $repository = $this->registry->get($repositoryName);
         $resource = $repository->get('/'.$path);
 
+        return $this->createResponse($resource);
+    }
+
+    /**
+     * There are two different changes that can currently be done on a cmf resource:
+     *
+     * - move
+     * - rename
+     *
+     * changing payload properties isn't supported yet.
+     *
+     * @param string $repositoryName
+     * @param string $path
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function patchResourceAction($repositoryName, $path, Request $request)
+    {
+        $repository = $this->registry->get($repositoryName);
+        $this->failOnNotEditable($repository, $repositoryName);
+
+        $resourceName = $request->get('node_name');
+
+        $targetPath = null;
+        if ($path !== $path) {
```

Comment:
Also, I think we should take http://williamdurand.fr/2014/02/14/please-do-not-patch-like-an-idiot/ into account. This way, we can already open this controller for more options (like editing), but at the moment we can add a guard to only handle `id` changes.